src/apps/chatter/components/chatbox/chatbox.js

```diff
@@ -1,12 +1,21 @@
 import './chatbox.less'
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
```

Comment:
I like to follow the python pep8 standard for ordering imports: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
✔️